yt_dlp/extractor/comedycentral.py

```diff
@@ -25,6 +25,9 @@ class ComedyCentralIE(MTVServicesInfoExtractor):
     }, {
         'url': 'https://www.cc.com/collection-playlist/cosnej/stand-up-specials/t6vtjb',
         'only_matching': True,
+    }, {
+        'url': 'https://www.cc.com/topic/a-clusterfunke-christmas',
```

Comment:
so what I was saying is do this:
```suggestion
        'url': 'https://www.cc.com/movies/tkp406/a-cluesterfuenke-christmas',
```

Reply:
Oh rip I copied the wrong url. Sorry about that and good catch.